superbit_lensing/medsmaker/scripts/process_mocks.py

```diff
@@ -38,14 +40,18 @@ def main():
     args = parser.parse_args()
     mock_dir = args.mock_dir
     outfile = args.outfile
+    outdir=args.outdir
     use_coadd = args.meds_coadd
     clobber = args.clobber
     source_selection = args.source_select
     select_stars = args.select_stars
+    data_dir = '/users/jmcclear/data/superbit/forecasting-analysis/'
```

Comment:
So here, `data_dir` is where the mask and weight files are stored. It was easier to have them be in a separate directory  because my mock data for any one full 3-hr cluster observation is spread out over a number of smaller directories each containing 2-6 exposures. Having copies of the mask and weights in each repo does take up storage space. Is there a way to make the argument optional?

Reply:
This is a reasonable plan. Given that mask and weight files will be stored in `mock_dir`, it makes sense to delete the `data_dir` usage and stick to `mock_dir`.